doc/fuzzing.md

```diff
@@ -150,6 +150,7 @@ $ cmake -B build_fuzz \
    -DCMAKE_CXX_COMPILER="$(brew --prefix llvm)/bin/clang++" \
    -DBUILD_FOR_FUZZING=ON \
    -DSANITIZERS=undefined,address,fuzzer \
+   -DAPPEND_LDFLAGS=-Wl,-no_warn_duplicate_libraries
```

Comment:
Reading the thread from the other PR, it's not clear if there is an upstream issue opne, or someone is going to open one. I see links to other discussions that look similar, but from what I can tell, aren't our issue.

Reply:
https://github.com/llvm/llvm-project/issues/102630